docs/source/release_notes/release-notes.rst

```diff
@@ -42,32 +42,51 @@ develop
     *    - Type
          - Change
 
-    *    - |changed|
-         - Increased the size of CleanNarrowTable used in benchmarks from 10,000 to 1,000,000 rows.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/1773>`__)
+    *    -
+         -
 
-    *    - |new|
-         - Added a benchmark ``KvsGetRowsBenchmarks`` for benchmarking the KVS ``getRows`` method.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/1770>`__)
+=======
+v0.38.0
+=======
 
-    *    - |fixed|
-         - Creating a postgres table with a long name throws if the truncated name (first sixty characters) is the same as that of a different existing table.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/1729>`__)
+6 Apr 2017
 
-    *    - |devbreak|
-         - The ``atlasdb-remoting`` project was removed. We don't believe this was used anywhere, but if you encounter any problems due to the project having being removed, please contact AtlasDB support.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/1750>`__)
+.. list-table::
+    :widths: 5 40
+    :header-rows: 1
+
+    *    - Type
+         - Change
 
     *    - |improved|
          - The default ``sweepBatchSize`` has been changed from 1000 to 100.
            This has empirically shown to be a better batch size because it puts less stress on the underlying KVS.
            For a full list of tunable sweep parameters and default settings, see :ref:`sweep tunable options <sweep_tunable_parameters>`.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1763>`__)
-       
+
+    *    - |fixed|
+         - Reverted `#1524 <https://github.com/palantir/atlasdb/pull/1524>`__, which caused dependency issues in upstream products.
```

Comment:
A note for 0.37 refers to "upstream products". A quick google search was [inconclusive](http://stackoverflow.com/questions/18401861/upstream-and-downstream-definition).